backend/utils/sidebar.py

```diff
@@ -34,7 +34,11 @@ async def __get_room_members(
                             }
         """
         room_members = room.get("room_members")
-        room_members.pop(member_id, "not-found")  # remove self from room members
+
+        if (
+            "".join(room_members.keys()) != member_id
+        ):  # checks if it's not a room with only the member_id
```

Comment:
It works @o4codes.
However, i don't know if this part is easily understandable in terms of readability.
That's my opinion. Let's wait for other opinions.
@pauline-banye @mykie88

Reply:
I understand your view, 
I'll see if it can be made more readable.
Though I feel this is quite readable because it's just a check to see if a combination of all member_ids in a room is still equal to the member_id making the request, if it's true then the member_id is the only one in that room.

Reply:
@zxenonx checkout the commit

Reply:
Yeah, I read the comment...
Thanks for trying to make the condition more readable, but I think the new changes you made is a bit more difficult to understand than the previous code.
Please, let's just revert to the previous commit.
Thanks.
@o4codes

Reply:
Frankly I think the present condition we have covers fully for any room that is not a personal DM. 
Therefore, this is more readable than the former ones, and the condition is more extensive to cover for what is not a personal DM.